NEWS.md

```diff
@@ -46,6 +46,8 @@
 -   `$min()` and `max()` now work on categorical variables (#868).
 -   New methods for the `list` subnamespace: `$n_unique()`, `$gather_every()` 
     (#869).
+-   Converts `clock_time_point` and `clock_zoned_time` objects from
+    the `{clock}` package to Polars datetime type (#861).
```

Comment:
```suggestion
-   `as_polars_series()` can now convert `clock_time_point` and `clock_zoned_time` objects from
    the `{clock}` package to Polars datetime type (#861).
```

Reply:
In fact, this is used internally in many different places, so it is better not to emphasize specific function.